mlflow/johnsnowlabs.py

```diff
@@ -0,0 +1,786 @@
+"""
```

Comment:
Can you format the code with black?

Reply:
resolved in https://github.com/mlflow/mlflow/pull/8556/commits/76d55dcdd647b6d183e933e3f7802995c95fc2fc